dlup/annotations.py

```diff
@@ -729,41 +746,53 @@ def from_darwin_json(
     def __getitem__(self, a_cls: AnnotationClass) -> SingleAnnotationWrapper:
         return self._annotations[a_cls]
 
-    def as_geojson(self, split_per_label: bool = False) -> GeoJsonDict | list[tuple[str, GeoJsonDict]]:
+    def as_geojson(self) -> GeoJsonDict:
         """
-        Output the annotations as proper geojson.
-
-        Parameters
-        ----------
-        split_per_label : bool
-            If set will return a list of a tuple with str, GeoJSON dict for this specific label.
+        Output the annotations as proper geojson. These outputs are sorted according to the `AnnotationSorting` selected
+        for the annotations. This ensures the annotations are correctly sorted in the output.
 
         Returns
         -------
         list of (str, GeoJsonDict)
         """
-        jsons = [(label, self[label].as_json()) for label in self.available_labels]
-        if split_per_label:
-            per_label_jsons = []
-            for label, json_per_label in jsons:
-                per_label_data: GeoJsonDict = {
-                    "type": "FeatureCollection",
-                    "features": [],
-                    "id": None,
-                }
-                for idx, json_dict in enumerate(json_per_label):
-                    per_label_data["features"].append(json_dict)
-                    per_label_data["id"] = str(idx)
-                per_label_jsons.append((label, per_label_data))
-            return per_label_jsons
+        coordinates, size = self.bounding_box
+        region_size = (coordinates[0] + size[0], coordinates[1] + size[1])
+        all_annotations = self.read_region((0, 0), 1.0, region_size)
```

Comment:
this could also be changed to the slide bounds (which uses limit-bounds)

Reply:
I understand now that the bounding box represents the area in which annotations exist. Then one could also specify: 
 all_annotations = self.read_region(coordinates, 1.0, size) 
Or is there a reason for starting at the top left corner of the WSI?

Reply:
It doesn't really matter